src/schemas/40-10007/schema.js

```diff
@@ -13,61 +13,60 @@ _.merge(modifiedToursOfDuty, {
       serviceBranch: {
         // All branches active during WWII and later as returned by EOAS getBranchesOfService
         'enum': [
-          'AC', // US ARMY AIR CORPS
-          'AF', // US AIR FORCE
-          'AR', // US ARMY
-          'CG', // US COAST GUARD
-          'CV', // CIVILIAN WAKE ISLAND NAS
-          'FP', // CIVILIAN FERRY PILOT
-          'MM', // US MERCHANT MARINE
-          'PH', // US PUBLIC HEALTH SERVICE
-          'NN', // NAVY NURSE CORPS
-          'WA', // WOMEN'S ARMY AUX CORPS
-          'WS', // WOMEN'S ARMY CORPS
-          'CF', // ROYAL CANADIAN AIR FORCE
-          'RO', // ROTC OF ARMY NAVY OR AF
-          'CA', // US CITIZEN WHO SERVED W/ALLIES
-          'WR', // WOMEN'S RESERVE OF NAVY,MC,CG
-          'CS', // CIVILIAN W/STRATEGIC SVC (OSS)
-          'KC', // QRTRMASTER CORPS KESWICK CREW
-          'CB', // DEFENSE OF BATAAN
-          'CO', // US ARMY TRANSPORT SERVICE
-          'CI', // CIV ID FRIEND/FOE (IFF) TECH
+          'AL', // ALLIED FORCES
           'CC', // AMERICAN FIELD SERVICE
```

Comment:
There was previously duplicated 'American Field Service' labels ('CC' and 'FS'). Should the duplicate be removed?

Reply:
i think it probably shouldn't be removed unless a stakeholder says it should be

Reply:
I agree. We should ask about it.

Reply:
Sounds good, in that case, it would be good to ask them how the labels should differ. This removal has been reverted.